### ISSUE TITLE
Feature Requests - Exceptions, Presets, Export, Import

### ISSUE THREAD

**Issue by CrackHub** [dessant/clear-browsing-data#60]
**Is your feature request related to a problem? Please describe.**

I'm really lookin for good cleaner. I found one but it is paid so looking for free alternatives. This extension good but a lots of features missing.

**Describe the solution you'd like**

**Exceptions -** This will allow us to protect web site datas. Example App settings (Twitch extensions etc)
**Presets -** We can have different setups
**Export, Import -** Share settings with others.

**Describe alternatives you've considered**

Currently paid extension have this features.

**Additional context**

![1](https://github.com/user-attachments/assets/593382fb-22f5-416f-a272-684f23441668)

![2](https://github.com/user-attachments/assets/55d97ff4-729d-4b78-8f8d-b8d7f8881c9d)

**Comment by dessant** [comment]
Yeah, those are nice features, and it wouldd definitely be awesome for someone to work on them and offer it all for free.

**Comment by dessant** [comment]
https://github.com/dessant/clear-browsing-data/issues/29
